libs/@local/eslint-config/legacy-base-eslintrc-to-refactor.cjs

```diff
@@ -109,7 +109,7 @@ module.exports = {
     "default-case": "off",
     "class-methods-use-this": "off",
     "react/no-unescapted-entities": "off",
-    "jsx-a11y/no-autofocus": "off",
+    "jsx-a11y/no-autoFocus": "off",
```

Comment:
The name of the rule is `"jsx-a11y/no-autofocus"`, which is why an eslint error has appeared in the block – we were previously turning the rule off. This change means we are _not_ turning the rule off – we are turning off a rule that doesn't exist. If we revert this change we can remove the eslint disable line added.
```suggestion
    "jsx-a11y/no-autofocus": "off",
```

Reply:
Ah that makes sense, thanks for catching this. Have addressed in https://github.com/hashintel/hash/pull/3434/commits/d5244c63f3f80232b507261855b0452297223176